{"componentChunkName":"component---src-templates-book-js","path":"/books/dotnet/fundamentals/c/beginner/exception/","result":{"data":{"markdownRemark":{"html":"<h1>Exception Handling</h1>\n<p>coming soon...</p>","frontmatter":{"slug":"dotnet/fundamentals/c/beginner/exception","stack":"C#","title":"Exception Handling","next":"dotnet/fundamentals/c/beginner/oops","prev":"../"},"id":"b4aedee6-96dd-54ee-af8d-341f796bfcd2","tableOfContents":"<ul>\n<li><a href=\"#exception-handling\">Exception Handling</a></li>\n</ul>","timeToRead":1},"allDirectory":{"edges":[]}},"pageContext":{"slug":"dotnet/fundamentals/c/beginner/exception","breadcrumb":{"location":"/books/dotnet/fundamentals/c/beginner/exception/","crumbs":[{"pathname":"/","crumbLabel":"Root"},{"pathname":"/books/","crumbLabel":"books"},{"pathname":"/books/dotnet/","crumbLabel":"dotnet"},{"pathname":"/books/dotnet/fundamentals/","crumbLabel":"fundamentals"},{"pathname":"/books/dotnet/fundamentals/c/","crumbLabel":"c"},{"pathname":"/books/dotnet/fundamentals/c/beginner/","crumbLabel":"beginner"},{"pathname":"/books/dotnet/fundamentals/c/beginner/exception/","crumbLabel":"exception"}]}}},"staticQueryHashes":["2948616848"],"slicesMap":{}}